src/main/java/it/gov/pagopa/pu/send/repository/SendNotificationNoPIIRepositoryExtImpl.java

```diff
@@ -109,4 +110,13 @@ public Optional<SendNotificationNoPII> findByIdAndOrganizationId(String notifica
 
     return Optional.ofNullable(mongoTemplate.findOne(query, SendNotificationNoPII.class));
   }
+
+  @Override
+  public Optional<SendNotificationNoPII> findByOrganizationIdAndIUV(Long organizationId, String iuv) {
+    Query query = new Query();
+    query.addCriteria(Criteria.where(Fields.organizationId).is(organizationId)
+      .and(FIELD_PAYMENT_NOTICE_CODE).is("3"+iuv));
```

Comment:
move this into a common utility method
```suggestion
      .and(FIELD_PAYMENT_NOTICE_CODE).is(iuv2nav(iuv)));
```